lib/flex_commerce_api/api_base.rb

```diff
@@ -111,7 +111,10 @@ def public_attributes
     end
 
     def meta_attribute(key)
-      attributes[:meta_attributes][key][:value] rescue nil
+      case attributes[:meta_attributes][key][:data_type]
+      when "related-files", "related-products" then self.send(key)
```

Comment:
now we have 3 more related data types : 'related-categories', related-static_pages', and 'related-resources'. Do we need to support them too?